packages/l/libmysofa/xmake.lua

```diff
@@ -0,0 +1,25 @@
+package("libmysofa")
+    set_homepage("https://github.com/hoene/libmysofa")
+    set_description("Reader for AES SOFA files to get better HRTFs")
+    set_license("BSD-3-Clause")
+
+    add_urls("https://github.com/hoene/libmysofa/archive/refs/tags/$(version).tar.gz",
+             "https://github.com/hoene/libmysofa.git")
+
+    add_versions("v1.3.2", "6c5224562895977e87698a64cb7031361803d136057bba35ed4979b69ab4ba76")
+
+    add_patches("v1.3.2", "patches/v1.3.2/fix-build.patch", "beb93eaaea535ca69669f5f77dbf96e42ab35e66f25c6889ac1a53c56da30528")
+
+    add_deps("cmake", "zlib")
+
+    on_install(function (package)
+        os.rm("share/default.sofa")
+        os.cp("share/MIT_KEMAR_normal_pinna.sofa", "share/default.sofa")
+        local configs = {"-DBUILD_TESTS=OFF"}
+        table.insert(configs, "-DBUILD_SHARED_LIBS=" .. (package:config("shared") and "ON" or "OFF"))
```

Comment:
Missing build_type `table.insert(configs, "-DCMAKE_BUILD_TYPE=" .. (package:is_debug() and "Debug" or "Release"))`